### PR TITLE
kvstore table migration during consensus upgrade

### PR DESCRIPTION
**Pull request by ahangsu**
## Summary

The support for #3890 on `kvstore` table migration during consensus upgrade, by upgrading `accountDBVersion` from version 7 (320 rounds PR) to version 8.

### CODE REVIEW

**Review by algorandskiy — COMMENTED** [review]
I do not see where a new box table gets created

File: ledger/accountdb.go
Comment (ahangsu) [reply ledger/accountdb.go:962]: ok, I will revert back the code change

File: ledger/trackerdb.go
Comment (ahangsu) [inline ledger/trackerdb.go:171]: accountDBVersion 6 -> 7 is reserved for 320 rounds.

DO NOT MERGE THIS PR UNTIL master MERGED INTO feature/avm-box

File: ledger/acctupdates.go
Comment (ahangsu) [reply ledger/acctupdates.go:1031]: I agree that it is weird to have the condition, but the case is, if we do not apply such condition, then preparing boxes DB queries would error, for there is no kvstore in database yet.

I assume this is not the best solution in `acctupdates.go`, but for `accountdb_test.go`, where we do `accountsInitTest`, I guess we can just drop the condition and create all the tables, if there is no ledger available.

File: ledger/trackerdb.go
Comment (ahangsu) [reply ledger/trackerdb.go]: ok, fixed after 320 in feature/avm-box and updated feature/avm-box merged in

**Review by jannotti — APPROVED** [review]
brilliant

**Pull request merged**
